tests/src/test/scala/whisk/core/container/test/ContainerUtilsTests.scala

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.container.test
+
+import org.junit.runner.RunWith
+import org.scalatest.fixture.FlatSpec
+import org.scalatest.junit.JUnitRunner
+import java.io.File
+
+import common.StreamLogging
+
+import whisk.common.TransactionId
+import whisk.core.container.ContainerUtils
+import whisk.common.Logging
+import whisk.core.container.ContainerHash
+import org.scalatest.BeforeAndAfter
+import java.io.FileWriter
+import java.nio.charset.StandardCharsets
+import org.scalatest.Matchers
+import org.scalatest.concurrent.TimeLimitedTests
+import org.scalatest.time.SpanSugar._
+
+/**
+ * Unit tests for ContainerPool and, by association, Container and WhiskContainer.
+ */
+@RunWith(classOf[JUnitRunner])
+class ContainerUtilsTests extends FlatSpec
+    with BeforeAndAfter
+    with Matchers
+    with StreamLogging
+    with TimeLimitedTests {
```

Comment:
Just discussed with @markusthoemmes - removing the time limit because it isn't required at the moment.